.github/ISSUE_TEMPLATE/magazine_edit.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Add a topic to collection of common questions, issues and mistakes / 編集会議ネタの新規項目追加
```

Comment:
@himorin I don't really understand how this is different from using 'Other'.

I checked the Japanese translation, and that seems to say something different from the English name (Addition of new items for editorial meeting material), which might make the distinction you were aiming at, but i couldn't get that from the English.